test/interim_etl_test.py

```diff
@@ -7,24 +7,6 @@
 logger = logging.getLogger(__name__)
 
 
-def test_interim_eia861_etl(pudl_settings_fixture, pudl_datastore_fixture):
```

Comment:
did you move these interim tests over into fast tests because these were not being run during the fast tests?

I agree that these should be run during the fast tests but also it seems real weird to have this module at all if the bulk of these tests have been removed and "interim" tests are now in two places

Reply:
I moved these over to the output tests because the code that connects the output object to the ETL functions was broken, and was not getting tested in the other arrangement, but to test that code the etl methods which are part of the output object need to be run, and that necessitates having an output object, and the output object fixture only exists in the output tests.

Reply:
that makes sense, but it seems odd to keep this interim model is most of the interim tests are removed from the module.

Reply:
We *could* remove this test (and the `interim_etl_test` module) if we wanted to, since the `pudl.output.ferc714` tests are using the Census data, and so are calling the tested function. The only difference would be that the `tract` and `state` level geometries wouldn't be getting pulled out of the GeoDatabase, but the functionality would be tested overall.